extensions/notebook/src/integrationTest/notebookIntegration.test.ts

```diff
@@ -127,6 +127,6 @@ function writeNotebookToFile(pythonNotebook: INotebook): vscode.Uri {
 async function ensureJupyterInstalled(): Promise<void> {
 	let jupterControllerExports = vscode.extensions.getExtension('Microsoft.sql-vnext').exports;
 	let jupyterController = jupterControllerExports.getJupterController() as JupyterController;
-	await jupyterController.jupyterInstallation.installReady;
+	await jupyterController.jupyterInstallation.startInstallProcess(false);
```

Comment:
Since these changes affect the integration tests, did you confirm that the tests still pass after this change?

Reply:
The local notebook test passed, but the remote test was failing during connection. Something about couldn't find "onReady" of undefined in the connection service.

Reply:
Removed that connection test, since we no longer support knox connections.